commercetools/commercetools-sdk-java-api/src/test/java-generated/com/commercetools/api/client/resource/ByProjectKeyApiClientsByIDTest.java

```diff
@@ -1,82 +0,0 @@
-
-package com.commercetools.api.client.resource;
```

Comment:
Can you please check why these tests are deleted in your branch?

Maybe rebase/cleanup. Seems to be a merge conflict

Reply:
I deleted them I will generate them back once I will finish with the PR.
If you put all the generated files you cannot see the files which are manually written